test/timecode/timecode_property_test.exs

```diff
@@ -139,10 +139,19 @@ defmodule Vtc.TimecodeTest.Properties.ParseRoundTripDrop do
   property "59.94 frames round trip" do
     check all(frames <- StreamData.integer(-5_178_816..5_178_816)) do
       assert {:ok, timecode} = Timecode.with_frames(frames, Rates.f59_94_df())
-      assert Timecode.frames(timecode) == frames
+      Timecode.frames(timecode) == frames
 
-      timecode_str = Timecode.timecode(timecode)
-      assert {:ok, ^timecode} = Timecode.with_frames(timecode_str, Rates.f59_94_df()), "#{frames}"
+      assert {:ok, ^timecode} = Timecode.with_frames(frames, Rates.f59_94_df())
+    end
+  end
+
+  property "rounded seconds is valid with roudning off" do
```

Comment:
```suggestion
  property "rounded seconds is valid with rounding off" do
```

Reply:
In f42af99f3dd475007652984b0a2f9f5ca43c0b3f